### PR TITLE
Initial Frequently Asked Questions (FAQs) Page (Issue #246)

### PR DESCRIPTION
**Pull request by mannybecerra**
Hey Team,

This is the initial creation of the requested Frequently Asked Questions (FAQs) page for issue #246. 

### Summary
* The FAQ page is a child of the `Documentation` page and linked to accordingly. 
* The slug/route of the FAQ page is `/documentation/frequently-asked-questions` (relative to the app's root)
* This new page is also cross-referenced and linked to from the [Contributing Guidelines docs](https://github.com/selfdefined/web-app/blob/prod/CONTRIBUTING.md)

**Notes:** 
* This page went through two noticeable iterations: the first PR changes, followed by a refactor of the FAQ page for leveraging `dl`, `dt` & `dd` markup tags for the FAQ content. Context can be found in our Slack channel [here](https://selfdefined.slack.com/archives/CHUUWHZGU/p1598747015096800) and [over here](https://selfdefined.slack.com/archives/C013RC20LNB/p1598818157017400?thread_ts=1598746993.005300&cid=C013RC20LNB) 
* @tatianamac in the thread under #246 , I've inferred that you may have more content around FAQs that you've collected ala DMs and other channels. If you would like for me to help get any of that other content worked into this new page, even under sub-categories, just lemme know. 

If changes are desired before merging this PR, I'm on standby to review and implement any requests. 

Gracias 🙌🏼

##### Lighthouse results (on local environment)

Screen shot of Lighthouse profile and performance results

##### New FAQs Page

Screen shot of new Frequently Asked Questions (FAQs) page under Documentation

##### Updated Documentation Page

Screen shot of the existing Documentation page for Self-Defined app updated to link to the new Frequently Asked Questions page

##### Updated Contributing Guidelines Doc

Screen shot of the existing Contributing Guidelines doc updated to cross-reference the new Frequently Asked Questions page

### CODE REVIEW

File: 11ty/documentation/index.md [hunk 1]
```diff
@@ -5,5 +5,6 @@ title: Docs
 As always, documentation is a work in progress. However, this is what we have so far:
 
 - [Examples](/documentation/examples/): Grab-and-go definition examples
+- [FAQs](/documentation/frequently-asked-questions): Answers to frequently asked questions
 - [Front Matter](/documentation/front-matter/): Configuration possibilities through front matter
 - [Development & Build](/documentation/development/): Notes for developing the site
```
Comment (tatianamac) [inline 11ty/documentation/index.md]:
```suggestion
- [Development & Build](/documentation/development/): Notes for developing the site
- [Front Matter](/documentation/front-matter/): Configuration possibilities through front matter
- [Examples](/documentation/examples/): Grab-and-go definition examples
- [FAQs](/documentation/frequently-asked-questions): Answers to frequently asked questions
```

File: 11ty/documentation/frequently-asked-questions.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  title: Documentation
+  href: '/documentation'
+---
+
+Answers to frequently asked questions.
+
+## When should I open an issue versus a pull request (PR)?
+
+### Open PR
+
```
Comment (tatianamac) [inline 11ty/documentation/frequently-asked-questions.md]:
```suggestion
Generally speaking, if you know how to open a PR in order to fix the issue, doing that is best. However, we recognise that it requires knowledge of GitHub and code to some degree, which is a form of gatekeeping. (We're currently working on ways to improve/remove this barrier to entry. We also are always willing to help if you'd like to learn. Reach out via our [Slack community](https://join.slack.com/t/selfdefined/shared_invite/zt-fczgm8b6-8ZZgHvLutNDXo~NjwaL7Iw)
```

File: 11ty/documentation/frequently-asked-questions.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+* Fix broken links
+* Add definitions/nuance/translations to words that exist on the site already but are undefined/partially defined/could use more nuance
+* Resolve issues that already exist
+
+### Open Issue
+
+* Concern or question around a term that you don't have clear prescriptive feedback around
+* Adding a new term that is not yet listed (with or without definition)
+* Adding/enhancing a feature that you're unsure is desired
+
```
Comment (tatianamac) [inline 11ty/documentation/frequently-asked-questions.md]:
```suggestion
* Are not sure how to open a PR but have a suggestion
* Express concern or ask a question around a term that you don't have clear prescriptive feedback around
* Add a new term that is not yet listed (with or without definition)
* Add or enhance a feature that you're unsure is desired
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -81,6 +81,8 @@ For this tutorial we will take the word «Obsessive Compulsive Disorder», as it
 
 We have created some examples that you can use as the baseline for your work. Take a look at the [File Examples](https://www.selfdefined.app/documentation/examples/) section of our documentation.
 
+**Related:** [When should I open an issue versus a pull request (PR)?](https://www.selfdefined.app/documentation/frequently-asked-questions/#when-should-i-open-an-issue-versus-a-pull-request-(pr)%13F)
+
 ## Get Help
 
 If you've already tried reading through our [documentation](https://www.selfdefined.app/documentation/) and are stuck, we're here to help and ask your questions:
```
Comment (tatianamac) [inline CONTRIBUTING.md:88]: Please add a link to [join our Slack community](https://join.slack.com/t/selfdefined/shared_invite/zt-fczgm8b6-8ZZgHvLutNDXo~NjwaL7Iw) under the **Get Help** section:

**Review by tatianamac — APPROVED** [review]

**Comment by tatianamac** [comment]
@mannybecerra Excellent work! I provided some prescriptive edits using the suggestions functionality. Please take a look and let me know if I can clarify anything.

As for other questions, I think it'd be great to collect some questions from folks in the Slack community and I can start to answer them. Would you be willing to lead that?

**Comment by mannybecerra** [comment]
@tatianamac and team - linting is failing with this PR due to the question mark at the end of one of the FAQ sub-headings, `When should I open an issue versus a pull request (PR)?` 

Should I should modify the FAQ in reference so 
- it doesn't end in a question mark, e.g. `When should I open an issue versus a pull request (PR)? Here's what to consider`; or
- break the question out of a subheading and leverage a `dl`->`dt`->`dd` markup approach? 

I'm leaning toward the latter but would appreciate some guidance on what others think would be best here when it's convenient

Gracias

**Comment by tatianamac** [comment]
@mannybecerra I would prefer the latter, personally.

**Comment by mannybecerra** [comment]
> @mannybecerra I would prefer the latter, personally.

@tatianamac sweet. I just saw your other comment about the prescriptive edits before I posted my original linting question to you. After seeing it, I was just going to move my question to your other question's thread to keep it all intact but now I just created a whirlwind. I'll update the FAQs content to leverage `dl` markup and take a look at your suggestions as well. 

Regarding `I think it'd be great to collect some questions from folks in the Slack community and I can start to answer them. Would you be willing to lead that?`, you bet. Just so I have some direction on what may work best for you on this front, do you have an approach that may work best for you in me leading this or do you want me to just run with it and we'll fine tune things along the way?

**Comment by tatianamac** [comment]
I trust you to take whatever approach makes the most sense with this. Mostly, I think there are valuable insights from the community built there. I think it could empower folks who have found our documentation confusing or unclear to get started.

Re: edits: Sounds perfect! Thank you again!

**Comment by mannybecerra** [comment]
> I trust you to take whatever approach makes the most sense with this. Mostly, I think there are valuable insights from the community built there. I think it could empower folks who have found our documentation confusing or unclear to get started.
> 
> Re: edits: Sounds perfect! Thank you again!

Great @tatianamac , thanks. 

I am hitting a snag with with the FAQ page and converting the content to use `dl` markup; `11ty`'s preprocessor doesn't seem to recognize the extended syntax for `dl` as outlined here, https://www.markdownguide.org/extended-syntax/#definition-lists. I'll reach out to our Slack community to see if you and others have insight on this

**Comment by mannybecerra** [comment]
> @mannybecerra I would prefer the latter, personally.

I agree @tatianamac . Done https://github.com/selfdefined/web-app/pull/288/commits/e79374985b221362f9eada141eab7e8685db8280

**Review by tatianamac — APPROVED** [review]
🚀 LGTM!

**Comment by tatianamac** [comment]
@mannybecerra I'll merge in this PR and if you'd like to work further on questions, we can collect those for a new PR.

**Comment by mannybecerra** [comment]
> @mannybecerra I'll merge in this PR and if you'd like to work further on questions, we can collect those for a new PR.

Suena bien @tatianamac !

**Pull request merged**